02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -170,26 +170,285 @@ Here's what has been added to the above since then:
 
 ## Why is PureScript's Documentation Lacking and How Do We Improve It?
 
-- Philosophy of the language:
-    - avoid (success at all costs)
-    - wrong intepretation:
-    - right interpretation
-- Current priority of core contributors: adding powerful features via breaking changes
-    - wrong interpretation: they don't care
-    - right interpretation: want to improve language
-    - Why a `v1.0` hasn't been released
-- Core Contributors are spread thin, so they cannot respond quickly to things
-    - The '75 libraries' problem
-    - Entrusting library maintenance to others and supporting them (e.g. the `event-stream` incident)
-    - Just like reviewing code, reviewing documentation takes a lot of thought
-    - Hard to keep documentation momentum
-    - Which libraries are 'core libraries'?
-- No clearly-defined mutually-held community-wide goal
-    - A desire to help exists, but no one is coordinating resources strategically
-- What exactly are FP's "Best Practices?"
-    - They are not well-defined, are assumed, or are habits
-    - Knowledge silos exist, so it's hard to extract that info
-- Slack-based discussions do not persist
+These are the reasons. Each wil be covered more in-depth following this:
```

Comment:
It looks like this is a summary of the rest of the document, and yet there is also a summary at the end of the document. I had a brief read of the summary at the end of the document, and I don't immediately see the value it adds over this intro/summary here and the following detail sections, so I propose removing the summary at the end of this document. What do you think?

Reply:
I think I actually prefer the summary at the end of the document more than this one (at least, it adheres to the heart of the more narrative medium of the Context rather than a bullet-point medium). In some ways, I'd rather move the narrative-format to where the bullet-point-format is and merge the initial bullet-point-format into the latter narrative-format. Then, when we could link to each section as they appear in the narrative-format.

Reply:
Latest changes now include this.

Reply:
👍